avm/res/network/connection/tests/e2e/vnet2vnet/main.test.bicep

```diff
@@ -0,0 +1,74 @@
+targetScope = 'subscription'
```

Comment:
I wonder if this test provides any value beyond the default test that also tests VNET to VNET

Reply:
I would ask you if you agree to remove the entire vnet2vnet folder

Reply:
That's kinda my suggestion. So if you agree with me - go ahead @fabmas ;)